cfgov/cfgov/settings/base.py

```diff
@@ -123,9 +125,13 @@
     {
         'NAME': 'wagtail-env',
         'BACKEND': 'django.template.backends.jinja2.Jinja2',
-        'DIRS': [V1_TEMPLATE_ROOT, V1_TEMPLATE_ROOT.child('_includes'),
+        'DIRS': [
+            V1_TEMPLATE_ROOT,
+            V1_TEMPLATE_ROOT.child('_includes'),
             V1_TEMPLATE_ROOT.child('_layouts'),
-            PROJECT_ROOT.child('static_built')],
+            DATA_RESEARCH_TEMPLATE_ROOT,
```

Comment:
Question: thus far we haven't separated out the templates for different apps, as everything is under `cfgov/jinja2/v1`. For example, the careers pages (even though their backend code is in `jobmanager`) or calendar pages (even though their backend code is in `cal`). I can see arguments for both sides, although I think I lean towards centralizing everything under one directory, if only because satellite app templates will always be including core modules/atomic elements. Centralizing also reduces possibility for name conflict, e.g. the off chance that multiple apps have files at the same path, e.g. `v1/index.html` and `otherapp/index.html`.

I'd be curious how @cfpb/cfgov-frontends would lean on this issue.